### PR TITLE
2023 conference updates

### PR DESCRIPTION
**Pull request by leesheppard**
- Updated venue
- Updated dates and schedule
- Added photos 
- Updated README with more information

### CODE REVIEW

File: views/2023/social-events.haml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+.introduction
+  .container-full.mb-4
+    %h3.event-title An array of fun and varied activities to surround the conference!
+
+.events
+  .container-full
+    .row.mb-5
+      .col-lg-3.col-md-4
+        .avatar.social-event-avatar
+          %img{ src: '/2023/images/alto.jpg' }
```
Comment (keithpitty) [inline views/2023/social-events.haml]: Should this image be `kingpin.jpg`?

File: views/2023/social-events.haml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            %p This event is free for all conference attendees.
+          .location
+            Wednesday 19th February, 7pm-9:30pm <br />
+            %a{ href: 'https://goo.gl/maps/nMorFH1wGMq1mSUP8' }
+              %i.fa.fa-map-marker-alt
+              King Pin - Upstairs at Crown Casino - 8 Whiteman Street, Melbourne
+    .row.mb-5
+      .col-lg-3.col-md-4
+        .avatar.social-event-avatar
+          %img{ src: '/2023/images/boatbuilders-yard.jpg' }
```
Comment (keithpitty) [inline views/2023/social-events.haml:25]: I can see `boatbuilders-yard.jpg` in `assets/2023/images` but don't understand why it isn't loading.

**Comment by keithpitty** [comment]
@leesheppard the images should load now that I've moved them to `/public/2023/images`.

**Review by keithpitty — APPROVED** [review]
@leesheppard great job on getting the site to this stage. I think it's good enough as an MVP. 👍 

/cc @quintrino

File: views/2023/social-events.haml (hunk 1 above)
Comment (leesheppard) [reply views/2023/social-events.haml]: Yes, good catch thank you. I mustn't have committed it in the last round.

File: views/2023/social-events.haml (hunk 2 above)
Comment (leesheppard) [reply views/2023/social-events.haml:25]: loading for me now in my local.

**Comment by leesheppard** [comment]
Social events page
Screen Shot 2022-11-06 at 22 11 41

**Comment by leesheppard** [comment]
Location page

Screen Shot 2022-11-06 at 22 13 20

**Comment by leesheppard** [comment]
Screen Shot 2022-11-06 at 22 13 35

**Pull request merged**